quality-control/human-biomolecular-data.md

```diff
@@ -53,22 +68,72 @@ After assessing data quality, the next step is data preprocessing. Data preproce
 - **Standardization and Calibration**: Standardisation and calibration of the biomolecular analysis methods used are essential for ensuring the accuracy and reliability of the results. This includes the use of validated assays, appropriate positive and negative controls, and quality assessment metrics to monitor the performance of the assays.
 - **Reproducibility and Robustness**: Reproducibility and robustness of the biomolecular analysis methods used are crucial for ensuring the consistency and reliability of the results across different laboratories and researchers. Therefore, it is essential to evaluate the reproducibility and robustness of the methods used, including the variability of results obtained by different operators, equipment, and reagents.
 - **Data Quality Assessment and Reporting**: Data quality assessment and reporting are critical for ensuring the accuracy and reliability of the results and communicating the findings to stakeholders effectively. This includes implementing appropriate data quality control measures, such as quality assessment metrics and statistical analysis, to identify and correct potential sources of error and ensure the accuracy and reliability of the data. Additionally, clear and concise reporting of the results, including appropriate data visualisation and interpretation, can help communicate the findings effectively to stakeholders.
+- **Sequencing quality control**: Information and tools for the assessment of the quality of the sequencing data in the context of COVID-19 can be found in the publication by the COVID-19 Host Genetics Initiative (HGI) on the [Whole Exome/Genome Sequencing Analysis Plan](https://github.com/covid19-hg/covid19_sequencing?tab=readme-ov-file).
+- **Benchmarking studies**: To obtain comparative quality assessment of the applied laboratory and analytical procedure, the participation in benchmarking studies is advised. Benchmarking studies can be performed in different ways, depending on the scientific discipline. The quality of sample processing results of aliquots from the same sample across several laboratories can be compared and reasons for quality deviations can possibly be identified. Benchmarking bioinformatics workflows can be done e.g. by providing mock genomes to the study participants and identifying those workflows that perform best in in silico identification or discovery of the respective trait in question (e.g. a variant, deletion of certain length, etc.). In the field of COVID-19, several benchmarking studies have been performed, e.g. for the sequencing  of viral genomes (https://pubmed.ncbi.nlm.nih.gov/34308277/), or for benchmarking bioinformatics approaches for virus identification workflows (https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03236-4; https://www.nature.com/articles/s41598-023-31368-3; https://www.biorxiv.org/content/10.1101/2023.08.07.552334v2.full; https://www.sciencedirect.com/science/article/pii/S138665322100175X).
```

Comment:
Last comment, I promise! :) here these long urls look a bit off when rendered. 

Could you replace them with the author + date, and the DOI behind them? See example below. @jaimeozaez 

![image](https://github.com/user-attachments/assets/c0930e03-69d9-4af8-b468-da51301f65db)

Reply:
I will also tag @svarona I think this is the only blocker anymore